lib/Conf.ml

```diff
@@ -588,10 +588,11 @@ module Formatting = struct
     let names = ["break-struct"] in
     let all =
       [ Decl.Value.make ~name:"force" `Force
-          "$(b,force) will break struct-end phrases unconditionally."
+          "$(b,force) will break struct-end and sig-end phrases \
```

Comment:
The doc is quite hard to read now. What do you think of this change: https://github.com/ocaml-ppx/ocamlformat/compare/main...Julow:ocamlformat:2682-man

Reply:
I like your change.

Reply:
I have pushed your change